packages/shared/styles/components/molecules/SfComponentSelect.scss

```diff
@@ -1,189 +1,155 @@
 @import "../../helpers";
 .sf-component-select-option {
-  display: flex;
-  box-sizing: border-box;
-  padding: var(--component-select-option-padding, var(--spacer-sm));
-  @include border(--component-select-option-border, 1px 0 0 0, solid, var(--c-light));
-  background: var(--component-select-option-background);
-  color: inherit;
-  min-height: var(
-    --component-select-height,
-    calc(
-      var(--spacer-base) + var(--spacer-2xs) +
-      var(--component-select-label-font-size, var(--font-size--lg))
-    )
-  );
-  @include font(
-    --component-select-option-font,
-    var(--font-weight--light),
-    var(--font-size--sm),
-    1.2,
-    var(--font-family--primary)
-  );
-  &.is-active {
-    --component-select-option-background: var(--c-light);
-  }
-  @include for-desktop {
-    --component-select-option-padding: var(--spacer-xs);
-    &:hover {
-      --component-select-option-background: rgba(var(--c-gray-base), 0.1);
-    }
-  }
+    display: flex;
```

Comment:
would be good to have the same indentation as before :)

Reply:
fixed, I think so...